apiserver/facades/agent/deployer/deployer.go

```diff
@@ -272,7 +275,7 @@ func (d *DeployerAPI) SetStatus(ctx context.Context, args params.SetStatus) (par
 // embedded APIAddresser *without* bumping the facade version.
 // It should be blanked when this facade version is next incremented.
 func (d *DeployerAPI) ModelUUID() params.StringResult {
-	return params.StringResult{Result: d.st.ModelUUID()}
+	return params.StringResult{Result: d.modelUUID.String()}
```

Comment:
I wonder if anything calls this? We could just return an empty string. 

Check what 3.6 does.